packages/use-sync-external-store/src/useSyncExternalStore.js

```diff
@@ -16,7 +16,7 @@ import * as React from 'react';
 export const useSyncExternalStore = React.useSyncExternalStore;
 
 if (__DEV__) {
-  console.error(
+  console['error'](
```

Comment:
We should add comments with here with explainers like we do for other, similar callsites